vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/SequenceFactoryTest.java

```diff
@@ -0,0 +1,53 @@
+package edu.asu.diging.vspace.core.services.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.MockitoAnnotations;
+
+import edu.asu.diging.vspace.core.factory.impl.SequenceFactory;
+import edu.asu.diging.vspace.core.model.ISequence;
+import edu.asu.diging.vspace.core.model.ISlide;
+import edu.asu.diging.vspace.core.model.impl.Module;
+import edu.asu.diging.vspace.core.model.impl.Slide;
+import edu.asu.diging.vspace.web.staff.forms.SequenceForm;
+
+public class SequenceFactoryTest {
+
+    @InjectMocks
+    private SequenceFactory sequenceFactoryToTest = new SequenceFactory();
+    
+    @Before
+    public void init() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
since you are not using any mocks in this test class you can remove this line and the ```injectMocks``` annotation.

Reply:
?